tests/wpt/metadata/html/semantics/embedded-content/media-elements/user-interface/muted.html.ini

```diff
@@ -1,76 +1,76 @@
 [muted.html]
   [getting audio.muted (script-created)]
-    expected: FAIL
+    expected: PASS
```

Comment:
These test do not seem to be passing. The reason is because we are not effectively muting the media element when the `mute` attribute is present on the `video` or `audio` tag. To do that we need to check if the `muted` attribute is mutated [here](https://dxr.mozilla.org/servo/source/components/script/dom/htmlmediaelement.rs#1734). If the attribute is present, `mutation.new_value(attr)` is `Some`, so we can do something like this when the mutated attribute is `muted`:

```rust
self.SetMuted(mutation.new_value(attr).is_some())
```